src/common.browser/CertChecks.ts

```diff
@@ -311,31 +331,31 @@ export class CertCheckAgent {
             options.agent = agent;
         }
 
-        return new Promise<Buffer>((resolve: (value: Buffer) => void, reject: (error: string | Error) => void) => {
-            ocsp.utils.getAuthorityInfo(req.cert, ocspMethod, (error: string, uri: string): void => {
+        return new Promise<Buffer>((resolve: (value: Buffer) => void, reject: (error: string | Error) => void): void => {
+            ocsp.utils.getAuthorityInfo(req.cert as tls.DetailedPeerCertificate, ocspMethod, (error: string, uri: string): void => {
                 if (error) {
                     reject(error);
                     return;
                 }
 
-                const parsedUri: {[k: string]: any} = parse.default(uri);
-                parsedUri.path = parsedUri.pathname;
-                options = { ...options, ...parsedUri };
+                const url = new URL(uri);
+                options = { ...options, host: url.host, protocol: url.protocol, port: url.port, path: url.pathname};
```

Comment:
@rhurey Thoughts? (This removes the url-parse dependency, which breaks gulp. The url-parse repo now says "use the WHATWG URL API if possible", FWIW). Also, this passes OCSP tests.